code/modules/cm_marines/altitude_control_console.dm

```diff
@@ -20,9 +20,10 @@ GLOBAL_VAR_INIT(ship_temp, 0)
 GLOBAL_VAR_INIT(ship_alt, SHIP_ALT_MED)
 
 /obj/structure/machinery/computer/altitude_control_console
```

Comment:
You will want to implement ex_act to ignore explosions, and make it unslashable/unacidable/unbreakable unless there is an avenue to repair/build the console again. Also `deconstructible` too depending?

Reply:
This is still outstanding.